components/common/mou-list.json

```diff
@@ -0,0 +1,29 @@
+{
+  "collectionName": "components_common_mou_lists",
```

Comment:
The collectionName is still mou_lists. Have you self-review and tested this thoroughly?